panel/template/react/react.html

```diff
@@ -0,0 +1,188 @@
+	  {{ embed(roots.busy_indicator) | indent(6) }}
+	</div>
+	{% endif %}
+
+  </nav>
+
+  <div id="content" style="overflow-x: hide;">
+    {% if nav %}
+    <div class="sidenav" id="sidebar">
+      <ul class="nav flex-column">
+        {% for doc in docs %}
+        {% for root in doc.roots %}
+        {% if "nav" in root.tags %}
+        {{ embed(root) | indent(8) }}
+        {% endif %}
+        {% endfor %}
+        {% endfor %}
+      </ul>
+    </div>
+    {% endif %}
+
+    <div class="main" id="main" style=" height: 100vh; ">
+
+      <div id="mydiv" style="overflow-x: hide; overflow-y: hide"></div>  <!--  div for react grid layout container  -->
+      
+      <div id="pn-Modal" class="pn-modal header-adjust">
+      <div class="pn-modal-content">
+        <span class="pn-modalclose" id="pn-closeModal">&times;</span>
+        {% for doc in docs %}
+        {% for root in doc.roots %}
+        {% if "modal" in root.tags %}
+        {{ embed(root) | indent(10) }}
+        {% endif %}
+        {% endfor %}
+        {% endfor %}
+          
+
+      </div>
+	  </div>
+    </div>
+  </div>
+</div>
+
+<script>
+  function openNav() {
+    document.getElementById("sidebar").style.left = 0;
+    document.getElementById("main").style.marginLeft = "16.7vw";
+    document.getElementById("sidebar-button").onclick = closeNav;
+    var interval = setInterval(function () { window.dispatchEvent(new Event('resize')); }, 10);
+    setTimeout(function () { clearInterval(interval) }, 210)
+  }
+
+  function closeNav() {
+    document.getElementById("sidebar").style.left = "-16.7vw";
+    document.getElementById("main").style.marginLeft = 0;
+    document.getElementById("sidebar-button").onclick = openNav;
+    var interval = setInterval(function () { window.dispatchEvent(new Event('resize')); }, 10);
+    setTimeout(function () { clearInterval(interval) }, 210)
+  }
+
+  var modal = document.getElementById("pn-Modal");
+  var span = document.getElementById("pn-closeModal");
+
+  span.onclick = function() {
+    modal.style.display = "none";
+  }
+
+  window.onclick = function(event) {
+    if (event.target == modal) {
+      modal.style.display = "none";
+    }
+  }
+</script>
+
+
+<script type="text/babel">
+
+const divStyle = { borderRadius: '5px',
+                    margin: '50px' };  
+
+const ResponsiveGridLayout = ReactGridLayout.WidthProvider(ReactGridLayout.Responsive)
+
+   class MyFirstGrid extends React.Component {
+    constructor(props) {
+    super(props);
+  }
+
+
+
+  resize_layout(obj, old_, new_, p, e, element) {
+    //  window.Bokeh.index[new_['i'].split('_')[1]].resize_layout();
+    console.log(obj)
+    window.dispatchEvent(new Event("resize"))
+  }  
+
+  resize_layout_stop(obj, old_, new_, p, e, element) {
+    setTimeout(function(){
+        console.log("I am setTimeout", new_['i']);
+        window.dispatchEvent(new Event("resize"))
+    //    window.Bokeh.index[new_['i'].split('_')[1]].resize_layout();
```

Comment:
Could you also just clean up the leftover comments and log statements here?

Reply:
sorry, I'm not used to community programming

Reply:
No worries, happens to all of us. This is an amazing first contribution to a project!